ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -1139,10 +1156,122 @@ KdbpContextFromPrevTss(
     *pTss = Tss;
     Context->Eip = Eip;
     Context->Ebp = Ebp;
+#endif
     return TRUE;
 }
 #endif
 
+#ifdef _M_AMD64
+
+typedef struct _RUNTIME_FUNCTION *PRUNTIME_FUNCTION;
```

Comment:
(I suppose naively including winnt.h here is a no-no?)
I wonder whether it could be of interest to extract those defines and functions from the XDK's winnt_old.h and place them into a dedicated XDK header, that is then included here or generates another private header :thinking: Or, could they be placed in some NDK header?

Reply:
Either could work. XDK was actually designed to be able to include directly. I will have a look where it's best to be placed. I guess placing it in rtlfuncs should work in both cases.

Reply:
I put it into NDK.